validium_mode_example/Context.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Once we get closer to the mergeable solution, we'd expect the proper logical structure of the project (this applies to test code as well -- I assume this package will evolve to some form of integration test).